### PR TITLE
Fixed top scroll for route change.

### LINKED ISSUE #744: Frontend: Changing route doesn't scroll to top in the new page
Example:
1. Open home page and go to footer
2. In footer click on the `about us` link

You can see the page doesn't scroll to top. See the screenshot
screenshot from 2017-03-02 02 04 20

### PR DESCRIPTION
**Pull request by anweshknayak**
fixes #744 

@aka-jain @spyshiv please review

### CODE REVIEW

File: frontend/src/js/route-config/route-config.js [hunk 1]
```diff
@@ -458,6 +458,7 @@
         $rootScope.$on('$stateChangeSuccess', function(event, toState, toParams, fromState, fromParams) {
             // Save the route title
             $rootScope.pageTitle = $state.current.title;
+            $window.scrollTo(0, 0);
```
Comment (spyshiv) [inline frontend/src/js/route-config/route-config.js]: Add a comment telling that why this line is added.

File: frontend/src/js/route-config/route-config.js [hunk 2]
```diff
@@ -458,6 +458,7 @@
...
             // alert($rootScope.pageTitle)
```
Comment (spyshiv) [inline frontend/src/js/route-config/route-config.js]: Remove this `alert` statement.

**Comment by spyshiv** [comment]
LGTM :).
Just do the changes requested then i will merge this PR.

**Comment by aka-jain** [comment]
@spyshiv But I think PR is already there for this issue from @gauthamzz. Should I close that?

**Comment by anweshknayak** [comment]
@aka-jain @spyshiv changes done. Please review

**Comment by aka-jain** [comment]
Merging it.

**Pull request merged**